pom.xml

```diff
@@ -193,10 +193,10 @@
 	<groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.7.0</version>
-        <!-- Require the Java 7 platform. -->
+        <!-- Require the Java 8 platform. -->
         <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
+          <source>1.8</source>
+          <target>1.8</target>
         </configuration>
```

Comment:
Note the javase javadoc link can also be updated to 8.

Reply:
:+1: pushed in 607d9c4